microsoft-edge/devtools-guide-chromium/evaluate-performance/unminify.md

```diff
@@ -0,0 +1,96 @@
+   
+To see your original function names in the flame chart in the **Performance** tool, use the **Unminify** (![The Unminify button icon.](images/unminify-icon.png)) button.
+
+Source mapping is used in DevTools to keep your client-side code readable and debuggable.  By clicking the **Unminify** button in the Performance tool, you can use source maps to create an unminified version of the performance profile you recorded.
+
+An example of a performance profile flame chart, with hard-to-read function names:
+
+![The minified flame chart in the performance profile, with hard-to-read function names.](images/minified-perf-profile.msft.png)
+
+An example of a performance profile flame chart, with some more meaningful function names restored:
+
+![The unminified flame chart in the performance profile, with meaningful function names restored.](images/unminified-perf-profile.msft.png)
+
+When recording a performance profile in the **Performance** tool, the **Main** section of the profile displays detailed flame charts of JavaScript functions that were called during the recording.  
+
+Ideally, in this flame chart view, you want to see the meaningful function names that are shown in your original source code.  However, if your production code is compiled, minified, or bundled, the **Performance** tool initially displays the function names that are extracted from your transformed, _production_ code.  These production function names might be obscure and might not easily map to function names in your original source code.
+
+
+To unminify a performance profile, so that some more meaningful function names are shown in addition to the minified identifiers, follow the steps below.
+
+
+<!-- ====================================================================== -->
+## Step 1 - Set up source maps
+
+Set up source maps, to host your source maps in addition to your production code.  Host your source maps where the **Performance** tool can find them.
+
+For details, see [Map the processed code to your original source code, for debugging](/microsoft-edge/devtools-guide-chromium/javascript/source-maps).
+
+
+<!-- ====================================================================== -->
+## Step 2 - Record a performance profile
+
+1. In the **Performance** tool, click the **Record** button, to record a performance profile for the page you want to analyze.  For details, see [Record runtime performance](/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference).
+
+   ![Recording a performance profile.](../media/evaluate-performance-performance-record-highlight.msft.png)
+
+   A performance profile is recorded, in the **Performance** tool.
+
+1. In the **Performance** tool, click in the main area (the flame chart), and use the mouse wheel to zoom in.  Click and drag left and right.  Look at the function names in the flame chart.  Select a small section of the profile and look at the flame chart.  Note the timestamp range.
```

Comment:
Since it's not like we're giving readers a demo site, I don't think we need to be so specific about where to look at function names. It's their recording, we can just provide instructions on looking at the flame chart.

```suggestion
1. In the **Performance** tool, click in the main area (the flame chart), and use the mouse wheel to zoom in.  Click and drag left and right.  Look at the function names in the flame chart.
```